src/gui/properties/propertieswidget.cpp

```diff
@@ -521,8 +521,14 @@ void PropertiesWidget::openFolder(const QModelIndex &index, bool containing_fold
     output = proc.readLine().simplified();
     if (output == "dolphin.desktop")
       proc.startDetached("dolphin", QStringList() << "--select" << fsutils::toNativePath(absolute_path));
-    else if (output == "nautilus-folder-handler.desktop")
+    else if (output == "caja-folder-handler.desktop")
+      proc.startDetached("caja", QStringList() << "--no-desktop" << fsutils::toNativePath(absolute_path));
+    else if (output == "nautilus.desktop")
```

Comment:
I also think you should test for `nautilus-folder-handler.desktop` too for completeness' sake.